api/db/run-migrations.sh

```diff
@@ -22,7 +16,9 @@ trap finish EXIT
 envsubst < "$(dirname "${BASH_SOURCE}")/create_db.sql" > $CREATE_DB_FILE
 
 echo "Creating database if it does not exist..."
+# This command is run regardless, and we rely on the commands in CREATE_DB_FILE being idempotent.
 mysql -h ${DB_HOST} --port ${DB_PORT} -u root -p${MYSQL_ROOT_PASSWORD} < ${CREATE_DB_FILE}
```

Comment:
I want to break this script up into one that builds the database and one that runs a liquibase command. Even better if we can drive liquibase without Bash or Ruby.